conjure-java-core/src/test/java/com/palantir/conjure/java/types/SafetyEvaluatorTest.java

```diff
@@ -130,7 +131,7 @@ void testMapSafeKey() {
                 .types(object)
                 .types(SAFE_ALIAS)
                 .build();
-        ConjureDefinitionValidator.validateAll(conjureDef);
+        ConjureDefinitionValidator.validateAll(conjureDef, SafetyDeclarationRequirements.ALLOWED);
```

Comment:
this is because i bumped the conjure parser

Reply:
Ah, I was wondering about that 👍